### PR TITLE
Add a bit of extra logging to the dns test when it encounters errors.

### PR DESCRIPTION
**Pull request by roberthbailey**

### CODE REVIEW

**Comment by ixdy** [comment]
LGTM

File: test/e2e/dns.go [hunk 1]
```diff
@@ -119,14 +119,14 @@ func assertFilesExist(fileNames []string, fileDir string, pod *api.Pod, client *
...
+			if _, err := client.Get().
 				Prefix("proxy").
 				Resource("pods").
 				Namespace(pod.Namespace).
 				Name(pod.Name).
 				Suffix(fileDir, fileName).
-				Do().Raw()
-			if err != nil {
+				Do().Raw(); err != nil {
+				Logf("Unable to read %s from pod %s: %v", fileName, pod.Name, err)
```
Comment (ixdy) [inline test/e2e/dns.go]: might need a newline at the end?

Comment (ixdy) [inline test/e2e/dns.go]: oh, I guess that's not necessary. OK.

Comment (roberthbailey) [inline test/e2e/dns.go]: Logf already adds a new line (see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/util.go#L177).

**Comment by roberthbailey** [comment]
Shippable failure seems like a flake, but I don't see a button in their UI to kick the build and make it re-run. Did they remove that recently?

**Pull request merged**